File: U1/EXAMEN/BanquitoServer_Soap_DotNet_GR01/BanquitoServer_Soap_DotNet_GR01/DTOs/RespuestaCreditoDTO.cs
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace BanquitoServer_Soap_DotNet_GR01.DTOs
{
    /// <summary>
    /// DTO para respuesta de otorgar crédito
    /// </summary>
    [DataContract]
    public class RespuestaCreditoDTO
    {
        [DataMember]
        public bool Exito { get; set; }

        [DataMember]
        public string Mensaje { get; set; }

        [DataMember]
        public string Cedula { get; set; }

        [DataMember]
        public string NumeroCredito { get; set; }

        [DataMember]
        public decimal MontoCredito { get; set; }

        [DataMember]
        public int NumeroCuotas { get; set; }

        [DataMember]
        public decimal CuotaMensual { get; set; }

        [DataMember]
        public decimal TasaInteres { get; set; }

        [DataMember]
        public List<CuotaAmortizacionDTO> TablaAmortizacion { get; set; }
    }
}

File: U2/PROYECTO/BanquitoServer_Restfull_Java_GR01/src/main/java/ec/edu/monster/DTOs/RespuestaCreditoDTO.java
package ec.edu.monster.DTOs;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.math.BigDecimal;
import java.util.List;

/**
 * DTO para respuesta de otorgar crédito
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class RespuestaCreditoDTO {
    private Boolean exito;
    private String mensaje;
    private String cedula;
    private String numeroCredito;
    private BigDecimal montoCredito;
    private Integer numeroCuotas;
    private BigDecimal cuotaMensual;
    private BigDecimal tasaInteres;
    private List<CuotaAmortizacionDTO> tablaAmortizacion;
}
